src/linear_algebra/affine_space/affine_map.lean

```diff
@@ -346,6 +346,10 @@ begin
   rw [h, equiv.comp_surjective, equiv.surjective_comp],
 end
 
+@[simp] lemma bijective_iff_linear_bijective (f : P1 →ᵃ[k] P2) :
```

Comment:
```suggestion
@[simp] lemma linear_bijective_iff {f : P1 →ᵃ[k] P2} :
```
and can you rename the previous two lemmas similarly?

Reply:
Yes, thanks for the hint.